offload-build.in

```diff
@@ -81,12 +81,13 @@ fi
 
 archbuild_cmd=("${repo}${archbuild_arch:+-$archbuild_arch}-build" "$@")
 
-trap 'rm -rf $SRCPKGDEST' EXIT INT TERM QUIT
+trap 'rm -rf $TEMPDIR' EXIT INT TERM QUIT
 
 # Use a source-only tarball as an intermediate to transfer files. This
 # guarantees the checksums are okay, and guarantees that all needed files are
 # transferred, including local sources, install scripts, and changelogs.
-export SRCPKGDEST=$(mktemp -d)
+export TEMPDIR=$(mktemp -d --tmpdir offload-build.XXXXXXXXXX)
+export SRCPKGDEST="${TEMPDIR}"
```

Comment:
You don't need to quote `export foo=${variable}` or `export foo=$(subprocess)` assignments. But if you're going to, you should be consistent. :p 

Here you're doing both...

Reply:
consistency sounds like a good thing :P